apis/r/R/TileDBCreateOptions.R

```diff
@@ -31,7 +31,9 @@
   # Filter-related schema parameters
   dataframe_dim_zstd_level = 3,
   sparse_nd_array_dim_zstd_level = 3,
-  offsets_filters = list('DOUBLE_DELTA', 'BIT_WIDTH_REDUCTION', 'ZSTD'),
+  dense_nd_array_dim_zstd_level = 3,
+  offsets_filters = list("DOUBLE_DELTA", "BIT_WIDTH_REDUCTION",
+                         list("name"="ZSTD", "COMPRESSION_LEVEL"=-1)),
```

Comment:
Does ZSTD here need the compression level set? I thought without providing a compression level it should just default set this to -1.

Reply:
Do you recall what error you were seeing. Because in the C++ API the default for this argument is `R"(["DOUBLE_DELTA", "BIT_WIDTH_REDUCTION", "ZSTD"])"`, so we don't set the compression level there and it works.

Reply:
I get a segfault, no error message and if memory serves it comes from expecting a second array element that is not coming.

Reply:
For what it is worth it happens (when running all tests) when it comes to `test-reopen.R` so about a quarter through.